qiskit/visualization/matplotlib.py

```diff
@@ -350,7 +350,11 @@ def draw(self, filename=None, verbose=False):
         if not self._user_ax:
             from matplotlib import get_backend
 
-            if get_backend() in ["module://ipykernel.pylab.backend_inline", "nbAgg"]:
+            if get_backend() in [
+                "module://ipykernel.pylab.backend_inline",
+                "module://matplotlib_inline.backend_inline",
+                "nbAgg",
+            ]:
```

Comment:
I think the same if statement exists for all the matplotlib based visualizations, state, histogram, pulse, etc. We probably need to update it in all of them